.github/ISSUE_TEMPLATE/DesignDoc.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Design Doc tracking issue
+about: A standard template to follow when leading a project
+labels: Daily, NewFeature
+---
+
+## Proposal
+
+## Tasks
+
+- [ ] Post **Proposal (Problem/Solution)** in #expensify-open-source
+- [ ] Paste Proposal in the space above with a link to the Slack thread
+- [ ] Email `strategy@expensify.com` and paste in the Proposal
```

Comment:
Should we say something like "After the discussion in expensify-open-source has concluded and been approved (2/3 :+1:)"?